subsys/bootloader/Kconfig

```diff
@@ -105,6 +105,18 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
+config SB_NUM_MONOTONIC_COUNTERS
```

Comment:
Would it be more clear if the configuration was:
- Support monotonic counters?
IF YES 
- How many upgrades do you want to support?

Reply:
Not sold on this, since we will have to add a build error for enabled+size0. It's more noise. Is it confusing the way it is?

Reply:
Could we have the size depend on SUPPORT_MONOTONIC_COUNTER?

Reply:
Added as an open question

Reply:
> Could we have the size depend on SUPPORT_MONOTONIC_COUNTER?

We'd still need to handle enabled + size0.

Reply:
Decided offline to add separate ENABLE config.